docker/build_image.sh

```diff
@@ -7,4 +7,12 @@ docker build \
     -t vllm_metax:v0 \
     --build-arg VLLM_VERSION=v0.10.2 \
     --build-arg MACA_VERSION=3.2.1 \
+     .
+
+# BUILDX_EXPERIMENTAL=1 docker buildx debug --invoke /bin/bash --on=error build \
+#     --network host \
+#     -f docker/vllm_metax.Dockerfile \
+#     -t vllm_metax:v1 \
+#     --build-arg VLLM_VERSION=v0.10.2 \
+#     --build-arg MACA_VERSION=3.2.1 \
      .
```

Comment:
this line needs to be commented as well `.`  is a valid shell command so it causes error messages: 

```
./docker/build_image.sh: line 18: .: filename argument required
.: usage: . filename [arguments]
```